cmd/oapi-codegen/oapi-codegen.go

```diff
@@ -30,6 +30,7 @@ import (
 
 func errExit(format string, args ...interface{}) {
 	_, _ = fmt.Fprintf(os.Stderr, format, args...)
+	fmt.Println("")
```

Comment:
This unfortunately won't work, as it's not printing to `os.Stderr`

Reply:
True. Sorry for being careless about stdout vs stderr.